<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Competences</class>
 <widget class="QDialog" name="Competences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(0, 255, 150, 255));*/</string>
  </property>
  <widget class="QTableView" name="tableView_client">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>0</y>
     <width>501</width>
     <height>370</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>501</width>
     <height>370</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>510</width>
     <height>370</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Lister">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>370</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Lister les Competences existantes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Menu">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>370</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;


</string>
      </property>
      <property name="text">
       <string>       METTRE A JOUR LE NOM D'UNE COMPETENCE</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Ancienne Competence</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Nouveau nom:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="NouvNom">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_maj">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="maj">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>MISE A JOUR</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>441</width>
     <height>115</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;


</string>
      </property>
      <property name="text">
       <string>        AJOUTER UNE NOUVELLE COMPETENCE</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Nom de la comp√©tence:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="NouvCpt">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="ajouter">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>AJOUTER</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>441</width>
     <height>115</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;


</string>
      </property>
      <property name="text">
       <string>              SUPPRIMER UNE COMPETENCE</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Selectionner la Competence</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_sup">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="supprimer">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>SUPPRIMER</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="statut">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
